### ISSUE TITLE
Steal the Keys starts with felony when you get inside a vehicle, making it impossible to leave the vehicle

### ISSUE THREAD

**Issue by danielfs0** [OpenDriver2/REDRIVER2#107]
Steal the Keys has now a weird bug, which makes you start with felony as soon you get inside a vehicle. No cops are showing though, only roadblocks. The fact is that when you wreck the other car, you can't leave yours to get inside the other car because the game thinks that the police chase is still happening. See the .gif below.

I'm using the latest release, and I have Driver 1's music turned on if that helps.

![example](https://user-images.githubusercontent.com/84626973/119227366-bfb1c680-bb0d-11eb-8fb4-ef473eb0331f.gif)

**Comment by SoapyMan** [comment]
I can see texture mods are there too.

**Comment by danielfs0** [comment]
I see. I'm using Comando52Colton's fixed car lights mod. This bug is strange though, because sometimes happens and sometimes it doesn't, while having custom content on. I'll try checking if this happens without them.

**Comment by danielfs0** [comment]
Alright, I tried several times this mission both vanilla and modded. I didn't manage to gain felony on start like stated above.
I don't know what happened in the first place, but I guess that's not a big deal? Still weird though.

**Comment by SoapyMan** [comment]
This is probably are fixed by e3f17a9, just need testing as I can't reproduce it

**Comment by danielfs0** [comment]
Yeep. Can confirm this happens in the clean, base, game, not frequently, but it happens. But this time I also spotted a police car in the minimap before quickly despawning.

![example2](https://user-images.githubusercontent.com/84626973/119229975-8b90d280-bb1a-11eb-8f7c-6e84540897c8.gif)

**Comment by danielfs0** [comment]
I found how to break it intentionally. You fastforward in the menu until the first demo replay shows, then while keeping fastforward you skip it. Then you wait the second demo replay loading and do the same.
After that, you go quickly to the mission menu, load Steal the Keys and you'll have this bug. I suspect it's related to police vehicles being still loaded in mission where they're disabled.

I made a small video, showcasing this; go to 0:19 to see how to do it.
https://www.youtube.com/watch?v=BseU9z3g4mM

As I stated above, it happens even without fastforwarding. This only makes it spot it easier.
